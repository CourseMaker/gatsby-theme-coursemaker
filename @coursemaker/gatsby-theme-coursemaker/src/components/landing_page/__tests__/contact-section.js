import React from "react"
import renderer from "react-test-renderer"
import ContactSection from "../contact-section"
import landingPageDouble from "../__mocks__/landing-page-double";

describe("ContactSection", () => {
    it("renders correctly", () => {
        // Created using the query from Header.js
        const landingPageSample = landingPageDouble;
        const tree = renderer.create(<ContactSection landingPage={landingPageSample} />).toJSON()
        expect(tree).toMatchSnapshot()
    })
})

describe("ContactSection", () => {
    it("renders correctly with null landing page", () => {
        // Created using the query from Header.js
        const landingPageSample = null;
        const tree = renderer.create(<ContactSection landingPage={landingPageSample} />).toJSON()
        expect(tree).toMatchSnapshot()
    })
})

describe("ContactSection", () => {
    it("renders correctly with missing landing page fields", () => {
        // Created using the query from Header.js
        const landingPageSample = landingPageDouble;
        landingPageSample.initialCTA = "";
        const tree = renderer.create(<ContactSection landingPage={landingPageSample} />).toJSON()
        expect(tree).toMatchSnapshot()
    })
})