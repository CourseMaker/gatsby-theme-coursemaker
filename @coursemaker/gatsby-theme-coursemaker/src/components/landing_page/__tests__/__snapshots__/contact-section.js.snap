// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactSection renders correctly 1`] = `null`;

exports[`ContactSection renders correctly with missing landing page fields 1`] = `null`;

exports[`ContactSection renders correctly with null landing page 1`] = `null`;
